src/renderer/shared/api/translation/locales/en.json

```diff
@@ -1144,9 +1077,9 @@
       "proxiesTabTitle": "Delegated authorities (proxies)",
       "proxyDeposit": "Total proxy deposit",
       "proxyToControl": "to control",
-      "proxyTooltipWhatIsProxy": "A delegated authority is a type of account that acts as an intermediary or delegate for another account. Delegated authorities add an additional layer of control for your assets, allowing other accounts to control various operations for your account in a customizable way depending upon the access type.",
+      "proxyTooltipWhatIsProxy": "A delegated authority is a type of account that acts as an intermediary or delegate for another account.Delegated authorities add an additional layer of control for your assets, allowing other accounts to control various operations for your account in a customizable way depending upon the access type.",
```

Comment:
Are you sure you don't need a space here?

Reply:
good catch, we for sure need a space.